### ISSUE TITLE
dissoc-in variable args

### ISSUE THREAD

**Issue by decoursin** [weavejester/medley#23]
dissoc removes one or more keys from a map:
```
(dissoc {:foo 1 :bar 2 :baz 3} :foo :bar)
```
medley.core/dissoc-in takes only and exactly one key vector set:
```
(medley.core/dissoc-in {:a {:b 3 :c 4 :d 5}} [:a :b] [:a :c])
ArityException Wrong number of args (3) passed to: core/dissoc-in  clojure.lang.AFn.throwArity (AFn.java:429)
```

**Comment by weavejester** [comment]
I think this seems like a sound idea.

**Comment by weavejester** [comment]
Fixed by #25.
